engines/mysql.py

```diff
@@ -92,14 +93,18 @@ def table_exists(self, dbname, tablename):
                 self.existing_table_names.add((schema.lower(), table.lower()))
         return (dbname.lower(), tablename.lower()) in self.existing_table_names
 
-    def set_engine_encoding(self, encoding='ISO-8859-1'):
+    def set_engine_encoding(self):
         """Set MySQL database encoding to match data encoding
 
-        Defaults to latin1 and falls back to it if an unknown encoding is provided
-
+           Please update the encoding lookup table if the required encoding is not present.
         """
-        encoding_lookup = {'ISO-8859-1': 'latin1'}
-        db_encoding = encoding_lookup.get(encoding, 'latin1')
+
```

Comment:
I have assigned a local variable `encoding` in separate engines which will handle this. Also I had made changes in `__init__.py` which are not being reflected. I know that `__init__.py` makes the directory into packages. So am I missing something, please help? I have figured out the error that `__init__.py` is not compiling to make the directory into package. Am I missing something?